src/Compute/Compute/VirtualMachine/Config/NewAzureVMConfigCommand.cs

```diff
@@ -110,8 +110,8 @@ public class NewAzureVMConfigCommand : Microsoft.Azure.Commands.ResourceManager.
 
         [Parameter(
             ValueFromPipelineByPropertyName = true,
-            HelpMessage = "The priority for the virtual machine.  Only supported values are 'Regular', 'Low' and 'Spot'.")]
-        [PSArgumentCompleter("Regular", "Low", "Spot")]
+            HelpMessage = "The priority for the virtual machine. Only supported values are 'Regular', 'Spot' and 'Low'. 'Regular' is for regular virtual machine. 'Spot' is for spot virtual machine. 'Low' is also for spot virtual machine but is replaced by 'Spot'. Please use 'Spot' instead of 'Low'.")]
```

Comment:
`Low` and `Spot` are merged into one? I'm a little confused by this help message.

Reply:
Yes.  "Low" and "Spot" are essentially the same.  Originally, we have only "Low". However, for marketing reason, we were asked to change it to "Spot".  But, in order not to break existing users, we still support "Low" as well as "Spot".   We just want to ask users to use "Spot", since it is the official priority value for Spot VM.